spec/factories.rb

```diff
@@ -0,0 +1,11 @@
+FactoryBot.define do
+  factory :user do
+    email "user@example.com"
```

Comment:
You need to use 'sequences' for this, pls search for Sequences here https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md

You can't create > 1 users with the same email, this is the problem